app.rb

```diff
@@ -1,8 +1,23 @@
 require 'sinatra/base'
-
+require './lib/peep'
 class Chitter < Sinatra::Base
-  get '/test' do
-    'Test page'
+  
+  get '/' do
+    'Chitter'
+  end
+  
+  get '/peeps' do 
+    @peeps = Peep.all
+    erb :'index'
+  end
+
+  get '/peeps/new' do
+    erb :"new"
+  end
+
+  post '/peeps' do 
```

Comment:
I would try and keep your route naming consistent, as above you are getting '/peeps/new/' I would also post 'peeps/new' here as this post is creating a new peep.